openapi.yml

```
@@ -0,0 +1,92 @@
+openapi: '3.0.2'
+info:
+  title: hitotsumami
+  description: find recipe from some ingredients
+  version: '1.0'
+servers:
+  - url: https://api.server.test/v1
+paths:
+  /post-recipe:
+    post:
+      tags: 
+        - post
+      description: post recipe
+      requestBody:
+        content:
+          application/json:
+            schema: 
+              $ref: '#/components/schemas/PostRecipe'
+            example:
+              user_id: 1
+              name: "おつまみキャベツ"
+              ingredients: [{"name": "キャベツ", "quantity":"1/4"}]
+              description: "簡単にばくばく食べられます"
+              time_required_min: 5
+              seasonings: [{"name": "にんにく","quantity":"適量"},{"name": "塩","quantity":"適量"},{"name": "ごま油","quantity":"適量"}]
+              steps: [{"content":"キャベツを切る"},{"content":"ジップロックに切ったキャベツと調味料を全部入れて混ぜる"}]
+
+      responses:
+        '200':
```

[must] 新規リソース作成が成功した場合のレスポンスは 201 になります。
```suggestion
        '201':
```